test/integration/fast_output_test.py

```diff
@@ -25,6 +25,32 @@ def fast_out(pudl_engine, pudl_datastore_fixture):
     )
 
 
+@pytest.fixture(scope="module")
+def fast_out_filled(pudl_engine, pudl_datastore_fixture):
+    """A PUDL output object for use in CI with net generation filled."""
+    return pudl.output.pudltabl.PudlTabl(
+        pudl_engine,
+        ds=pudl_datastore_fixture,
+        freq="MS",
+        fill_fuel_cost=True,
+        roll_fuel_cost=True,
+        fill_net_gen=True,
+    )
+
+
+@pytest.fixture(scope="module")
+def fast_out_annual(pudl_engine, pudl_datastore_fixture):
```

Comment:
I would love to have a larger conversation about how we want to deal with all of the class level parameters we've been adding to `pudl_out`.... It feel haphazard right now. Which comes up here... I could imagine having an output fixture  where we are do all of our imputations but right now the imputation are sloshed around in both of the standard output and this new one. I'd love some of your input on this because these tests have mostly been your baby.